tools/logrotate_config/package_after_install_script.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Create the log file for gcsfuse package.
+mkdir -p /var/log/gcsfuse/
+
+# Installs logrotate configuration files.
+mkdir -p /etc/logrotate.d/
+
+cat > /etc/logrotate.d/gcsfuse <<EOF
+/var/log/gcsfuse/*.log {
+    rotate ${LOGROTATE_FILES_MAX_COUNT:-5}
```

Comment:
Pass max count as build_arg and convert into a environment variable - LOGROTATE_FILES_MAX_COUNT, same for LOGROTATE_MAX_SIZE.

Reply:
will these changes be done after this PR is merged?

Reply:
Nope, all the changes will go in the same PR.